src/constants/education.js

```diff
@@ -0,0 +1,13 @@
+const EDUCATION = {
+  p: 'Doctorate',
```

Comment:
These should probably map to message ids like "common.education.phd" etc.  I can fix it up after merging, though.